### PR TITLE
Nucleo G474RE - Timer Input Capture Example

### PR DESCRIPTION
**Pull request by StevenMacias**
Added example that generates a PWM signal and monitors it with a second timer configured as input capture.

### CODE REVIEW

File: examples/nucleo_g474re/timer_input_capture/main.cpp [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+
+	while (true)
+	{
+		modm::delay(100ms);
+
+		// If USER button is pressed:
+		if (!Button::read())
+		{
+			float periof_of_signal_ms = getPeriodOfSignalInMs();
```
Comment (salkinium) [inline examples/nucleo_g474re/timer_input_capture/main.cpp]:
```suggestion
			float period_of_signal_ms = getPeriodOfSignalInMs();
```

**Review by salkinium — COMMENTED** [review]
Very well documented, I like it!

**Comment by salkinium** [comment]
Oh, and please squash your commits of course. You can force push to your branch to overwrite this PR instead of opening a new one.

**Comment by StevenMacias** [comment]
> Very well documented, I like it!

Thank you! Now it should be squashed. 
Is it preferred in modm to have a single commit when merging branches? Thanks!

**Comment by salkinium** [comment]
> Is it preferred in modm to have a single commit when merging branches?

I prefer to have a clean commit history to help me write the changelog without too much effort. So we generally prefer to have a logical grouping of commits without any fixup commits inbetween and always up to date to avoid merge conflicts for non-breaking PRs. But it's mostly subjective to have a "nice" commit history.

**Review by salkinium — APPROVED** [review]

**Pull request merged**